### DISCUSSION TITLE
Why do Amazfit office confirm or reject developer’s watch faces too late?

### DISCUSSION THREAD

**Discussion by tubebell** [zepp-health/zeppos-forum#332] [Q&A]
Hi, I made several watch faces for Amazfit smartwatch. I've been waiting for a few days, but it's not getting confirmed or rejected. Is the process usually slow? My account id is [3303035400].

**Comment by andrejkurusiov** [comment]
Hi @tubebell I wish I could see instant review. If OrientalBW watch face for Amazfit GTS4 is yours, is there any space or source code to ask for minor adjustments, like DDMM format and battery status addition?

**Reply by tubebell** [reply]
Hi, I was so busy so I couldn't make additional version. Today I uploaded new version as you wish. Wait for a few days. Thanks.

**Reply by andrejkurusiov** [reply]
Thanks! Is there any way to find your watch faces, as in the app it's very random? Somehow many faces are done with only US regional setup in mind.. I would make an effort to adjust them if I could.

**Reply by tubebell** [reply]
Welll... sometimes I coundn't find my watchfaces easily, haha! New version of watchface's name is [OrientalBW] and face category is [FreeStyle] as same as old verision. Good luck!

**Reply by andrejkurusiov** [reply]
Late thanks :) Do you have the repo open somewhere so that I could make modifications the way I wanted to (it's hard to explain and needs some visual testing)?

**Reply by tubebell** [reply]
@andrejkurusiov Sorry, but I don’t have any personal repos. But if you want conceptual images about watchfaces, you can send me through this site. :)
